ingest/expression_files/expression_files.py

```diff
@@ -112,12 +115,27 @@ def check_unique_cells(cell_names: List, study_id, client):
             "$nor": [{"name": "All Cells"}],
         }
         # Returned fields from query results
-        field_names = {"values": 1, "_id": 0}
+        FIELD_NAMES = {"values": 1, "_id": 0}
+        COLLECTION_NAME = "data_arrays"
+
```

Comment:
I don't see the part of this method that handles checking unique values in non-raw-counts files.  Let's touch base today so you can talk me through the approach -- I'm probably missing something